test/import_test.js

```diff
@@ -13,7 +13,19 @@ function addTest(name, translatorID, input, expected) {
 
 		assert.equal(response.headers['zotero-translator-id'], translatorID);
 
-		assert.deepEqual(response.body, expected);
+		for (const items of [expected, response.body]) {
+			for (const item of items) {
+				delete item.key
+				delete item.version
+				delete item.parentItem
+				for (const array of ['tags', 'notes', 'attachments', 'seeAlso']) {
```

Comment:
`array` → `prop`

"array" is a bit of a confusing variable name even if it's not technically a reserved word.

(Also, FWIW, we mostly don't bother with `const`, which is verbose and fairly unnecessary, particularly in small blocks like these where there's really no change of its being rebound accidentally.)

Reply:
Can you add a comment to clarify that this is deleting empty arrays?

Reply:
Done. I'll change the const if you want -- I prefer const because I think it expresses content better, and I find the scoping rules for var very confusing.

Reply:
The scoping is great, but you get that with `let` without the extra verbosity, and the protections of `const` don't add much value in small blocks like this.